src/components/EditSections/EditExtendedInfo/DepartmentsNameEdit/DepartmentsNameEdit.test.js

```diff
@@ -63,3 +64,10 @@ describe('Given DepartmentsNameEdit', () => {
     expect(screen.queryByPlaceholderText(/ui-users.extended.department.default/i));
   });
 });
+
+describe('Given DepartmentsNameEdit with disabled: true', () => {
+  it('should not render add button', async () => {
```

Comment:
It renders it but the button is disabled, isn't it? Adjust the description please